apps/builddao/widget/components/modals/CreateProject.jsx

```diff
@@ -0,0 +1,304 @@
+  let filtered = contributors.map((contributor) =>
+    contributor.customOption ? contributor.label : contributor
+  );
+  setDistributors(filtered);
+};
+
+const Main = styled.div`
+  display: flex;
+  gap: 1rem;
+  padding: 1rem;
+
+  @media (max-width: 768px) {
+    flex-direction: column;
+    .lhs,
+    .rhs {
+      width: 100%;
+    }
+  }
+  .form-control {
+    background: transparent;
+  }
+  .lhs {
+    min-width: 400px;
+    > div {
+      width: 100%;
+    }
+  }
+  .rhs {
+    min-width: 400px;
+  }
+  .form-group {
+    width: 100%;
+    & > div > div.p-2 {
+      background: transparent !important;
+      border: 1px solid var(--border-color, #ccc);
+      border-width: 0px 1px 1px;
+      border-top-style: initial;
+      border-right-style: solid;
+      border-bottom-style: solid;
+      border-left-style: solid;
+      border-top-color: initial;
+      border-right-color: rgb(222, 226, 230);
+      border-bottom-color: rgb(222, 226, 230);
+      border-left-color: rgb(222, 226, 230);
+      border-image: initial;
+      border-bottom-left-radius: 0.375rem;
+      border-bottom-right-radius: 0.375rem;
+      min-height: 9em;
+      * {
+        color: var(--text-color, #fff);
+      }
+    }
+    .nav-link {
+      color: var(--text-color, #fff);
+    }
+  }
+  .form-check {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    padding-left: 0px;
+  }
+`;
+useEffect(() => {
+  console.log({
+    title,
+    description,
+    location,
+    contributors,
+    twitter,
+    gitHub,
+    telegram,
+    website,
+    selectedTabs,
+    tags,
+    avatar,
+    coverImage,
+  });
+}, [
+  title,
+  description,
+  location,
+  contributors,
+  twitter,
+  gitHub,
+  telegram,
+  website,
+  selectedTabs,
+  tags,
+  avatar,
+  coverImage,
+]);
+
+return (
+  <Modal
+    open={showModal}
+    title={"Create Project"}
+    onOpenChange={toggleModal}
+    toggle={toggle}
+  >
+    <Main>
+      <div className="lhs d-flex flex-column gap-4">
+        <InputField
+          key={"Project-Title"}
+          label={"Project Title"}
+          placeholder={"Enter Project Title"}
+          value={title}
+          onChange={(e) => setTitle(e.target.value)}
+        />
+
+        <div className="form-group">
+          <label className="mb-1">Description</label>
+          <TextEditor
+            value={description}
+            onChange={(e) => setDescription(e.target.value)}
+          />
+        </div>
+        <InputField
+          key={"Location"}
+          label={"Location"}
+          placeholder={"Enter location"}
+          value={location}
+          onChange={(e) => setLocation(e.target.value)}
+        />
+        <div className="form-group">
+          <label className="mb-1">Project Contributors</label>
+          <Typeahead
+            multiple
+            options={
+              allMainnetAddresses ?? ["frank.near", "ellie.near", "jane.near"]
```

Comment:
how many accounts does the social db returns?
I believe it will be a lot, may be we can show the followed accounts only

Reply:
Followed accounts is a great idea; yes let’s do it. Social.get(context.account/graph/follow/*)